<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<title>ADHD: The Patience</title>
<meta name="author" content="Dilip G"/>
<style type="text/css">
.underline { text-decoration: underline; }
</style>
<link rel="stylesheet" href="../revjs/dist/reveal.css"/>

<link rel="stylesheet" href="../revjs/dist/theme/dracula.css" id="theme"/>
</head>
<body>
<div class="reveal">
<div class="slides">
<section id="sec-title-slide">
<h1 class="title">ADHD: The Patience</h1><h2 class="author">Dilip G</h2><p class="date">Created: 2023-09-24 Sun 22:22</p>
</section>
<section id="table-of-contents-section">
<div id="table-of-contents" role="doc-toc">
<h2>Table of Contents</h2>
<div id="text-table-of-contents" role="doc-toc">
<ul>
<li><a href="#/slide-1">1. Introduction</a></li>
<li><a href="#/slide-2">2. 01 Emerging drugs for the treatment of attention-deficit hyperactivity disorder (ADHD)</a></li>
<li><a href="#/slide-3">3. 02 ADHD at the workplace: ADHD symptoms, diagnostic status, and work-related functioning</a></li>
<li><a href="#/slide-4">4. 03 Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains</a></li>
<li><a href="#/slide-5">5. 04 Effects of physical exercise on children with attention deficit hyperactivity disorder</a></li>
<li><a href="#/slide-6">6. 05 Examining Differences in the Genetic and Functional Architecture of ADHD Diagnosed in Childhood and Adulthood</a></li>
<li><a href="#/slide-7">7. 06 Genome-wide DNA methylation in association with ADHD, and in association with impulsive and callous traits</a></li>
<li><a href="#/slide-8">8. 07 Transcriptomic risk scores for attention deficit/hyperactivity disorder</a></li>
<li><a href="#/slide-9">9. 08 Genetics of attention deficit hyperactivity disorder</a></li>
<li><a href="#/slide-10">10. 09 Discovery of the first genome-wide significant risk loci for attention-deficit/hyperactivity disorder</a></li>
<li><a href="#/slide-11">11. 10 Comprehensive analysis of omics data identifies relevant gene networks for Attention-Deficit/Hyperactivity Disorder (ADHD)</a></li>
<li><a href="#/slide-12">12. 11 Epigenetics and Attention-Deficit/Hyperactivity Disorder: New Perspectives?</a></li>
<li><a href="#/slide-13">13. 12 Sleep disturbances in ADHD: investigating the contribution of polygenic liability for ADHD and sleep-related phenotypes</a></li>
<li><a href="#/slide-14">14. 13 Treatment biomarkers for ADHD: Taking stock and moving forward</a></li>
<li><a href="#/slide-15">15. 14 ADGRL3 genomic variation implicated in neurogenesis and ADHD links functional effects to the incretin polypeptide GIP</a></li>
<li><a href="#/slide-16">16. 15 Cry1Δ11 mutation induces ADHD-like symptoms through hyperactive dopamine D1 receptor signaling</a></li>
<li><a href="#/slide-17">17. Bibliography</a></li>
<li><a href="#/slide-18">18. Glossary</a></li>
</ul>
</div>
</div>
</section>

<section>
<section id="slide-1">
<h2 id="1"><span class="section-number-2">1.</span> Introduction</h2>
</section>
<section>
<ul>
<li>A Neuro-developmental disorder, chronic condition characterized by hyperactivity-impulsivity &amp; inattention.</li>
<li>Impairment in multiple domains of daily living.</li>
<li>Nosological system (DSM &amp; ICD) set the criteria to classify disease or disorder.</li>
<li>Prevalence of 5.8% in children &amp; adolescents, 2.5% of adults.</li>
<li>It can comorbid with other conditions.</li>
<li>There is no cure as such, but medication and treatment (CBT) are viable.</li>
<li>Treatments can be stimulant based or non-stimulant, but stimulants are said to be more effective.</li>
<li>Usual treatments target dopaminergic &amp; noradrenegic system.</li>

</ul>

</section>
</section>
<section>
<section id="slide-2">
<h2 id="2"><span class="section-number-2">2.</span> 01 Emerging drugs for the treatment of attention-deficit hyperactivity disorder (ADHD)</h2>
</section>
<section>
<ul>
<li>Aim was to study pharmacological treatment of ADHD made during 2010-2020.</li>
<li>During studies there were dropouts mainly due to side effects.</li>
<li>About 20% of treated patients could not tolerate their therapies or experience insufficient efficacy.</li>
<li><code>Amantadine</code> to enhance cognitive functions and <code>tipepidine</code> for hyperactivity/impulsivity. Stand-alone emerging treatments for ADHD include <code>viloxazine</code> and <code>dasotraline</code> (pediatric)</li>
<li>Monoaminergic (neurotransmitters) reuptake inhibitors have a great potential to improve ADHD symptoms.</li>

</ul>

</section>
</section>
<section>
<section id="slide-3">
<h2 id="3"><span class="section-number-2">3.</span> 02 ADHD at the workplace: ADHD symptoms, diagnostic status, and work-related functioning</h2>
</section>
<section>
<ul>
<li>Aim was to examine the nature of work-related problems and impairments of adults with ADHD, and neuropsychological test performance.</li>
<li>Sample size of 1231 individuals from workplace and other 134 adults diagnosed from ADHD clinic.</li>
<li>It was questionnaire &amp; Neuropsychological performance tests based evaluated using <code>Conners’ Adult ADHD Rating Scale (CAARS)</code>: 66 items, scaled on four-points and followed by <code>Weiss Functional Impairment Rating Scale (WFIRS)</code>: 70 items, scored on four-point scale.</li>
<li>Statistically Analysis concludes that work-related issues are common in individuals diagnosed with ADHD but also in individuals with symptoms of ADHD.</li>
<li>Individuals with ADHD experienced problems at work in particular in not meeting their own standards and perceived potential.</li>
<li><p>
Although some neuropsychological tests did not predict work-related problems.
</p>

</section>
<section></li>
<li>Comparison of the proportion of individuals indicating impairment (score ≥ 2) in each of the <code>WFIRS</code> Work item between patients with ADHD (n = 134) and the community comparison group (n = 268)
<img src="file:///home/idlip/d-sync/projects/adhd/work-prob.png" alt="work-prob.png" /></li>

</ul>

</section>
</section>
<section>
<section id="slide-4">
<h2 id="4"><span class="section-number-2">4.</span> 03 Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains</h2>
</section>
<section>
<ul>
<li>GWAS of 38,691 Individuals &amp; 186,843 controls with ADHD (largest uptake).</li>
<li>Data source from iPSYCH (25,895 cases;  37,148 controls), deCODE genetics (8,281 cases; 137,993 controls) and previously published  data from 10 ADHD cohorts with European ancestry collected by the PGC (4,515 cases; 11,702  controls),</li>
<li>Performed transcriptome-wide  association study (TWAS) of the genetically regulated gene expression using EpiXcan to identify risk genes on 924 samples.</li>
<li>TWAS identified 23 distinct genes with different gene expression level in ADHD compared to Control.</li>
<li>Identified <b>27 genome-wide significant loci</b>, double compared to previous analysis (<b>ADHD2019</b>).</li>
<li>Three of the strongest associated loci (<b>p &lt; 5x10<sup>-14</sup> )</b> were on chromosome 1, 5 and 11.</li>
<li>Observed that ADHD risk genes are significant in frontal cortex of brain in cells like inhibitory, excitatory and dopaminergic neuron.</li>
<li>Summarize that ADHD is polygenic and also is influenced by other psychiatric disorders.</li>

</ul>

</section>
</section>
<section>
<section id="slide-5">
<h2 id="5"><span class="section-number-2">5.</span> 04 Effects of physical exercise on children with attention deficit hyperactivity disorder</h2>
</section>
<section>
<ul>
<li>To Examine effects of physical exercise in children with ADHD.</li>
<li>Aerobic moderate–high intensity exercise can promote BNDF synthesis, enhance neuroplasticity, and decrease the risk of nervous system disorders.</li>
<li>Intervention of single-bout of exercise offers immediate and persistent benefits in cognitive function, and long-term exercise training can effectively improve overall fitness and exert long-lasting effects on cognitive function.</li>
<li>At early stage due to impulsiveness, lack of motivation, impatience, and persistence of children with ADHD, interval training (HIIT) is the most recommended form of exercise.</li>
<li>Emphasized on perceptual motor exercises combined with cognitive tasks to improve cognitive function of children with ADHD.</li>

</ul>

</section>
</section>
<section>
<section id="slide-6">
<h2 id="6"><span class="section-number-2">6.</span> 05 Examining Differences in the Genetic and Functional Architecture of ADHD Diagnosed in Childhood and Adulthood</h2>
</section>
<section>
<ul>
<li>Aim was to investigate whether individuals who are diagnosed as children differ from those who are diagnosed in adulthood with respect to shared and unique architecture at the genome-wide, functional, and gene expression level of analysis.</li>
<li>Used Genomic SEM (structural equation modeling) to investigate genetic correlations of GWAS sample taken from iPSYCH.</li>
<li>Then applied Stratified Genomic SEM and transcriptome-wide SEM (T-SEM) to identify functional annotations and patterns of gene expression associated with genetic risk sharing or divergence across the ADHD subgroups.</li>
<li>At the functional and gene expression levels, Stratified Genomic SEM and T-SEM analyses revealed three annotations and 22 genes, respectively, that were significantly associated with genetic risk sharing across the subtypes.</li>
<li>Adulhood diagnosed ADHD exhibited a significantly larger negative <b>r<sub>g</sub></b> (genetic correlation) with educational attainment.</li>
<li>The genetic correlation (<b>r<sub>g</sub></b>) across childhood and adulthood diagnosed ADHD was 0.76 (SEr = 0.06).</li>

</ul>

</section>
<section id="slide-6-1">
<h3 id="6-1"><span class="section-number-3">6.1.</span> T-SEM Pinpoints Genes Associated with General ADHD</h3>
<ul>
<li>Univariate TWAS  reports two significant gene expression associated with adulthood and 19 genes associated with childhood ADHD.</li>
<li>Miami plot for gene expression hits for the combined factor of childhood and adulthood diagnosed ADHD.</li>

</ul>
<p>
The upper and lower blue lines represent the Bonferroni corrected significance threshold. Genes surpassing the upper and lower cutoff respectively are upwardly and downwardly regulated respectively in the ADHD factor. The most significant gene across tissue types are labeled and colored as red dots.
<img src="file:///home/idlip/d-sync/projects/adhd/gene-exp-plot.jpg" alt="gene-exp-plot.jpg" />
</p>

</section>
</section>
<section>
<section id="slide-7">
<h2 id="7"><span class="section-number-2">7.</span> 06 Genome-wide DNA methylation in association with ADHD, and in association with impulsive and callous traits</h2>
<ul>
<li>To study DNA <b><span class="underline">methylation</span></b> in persistent ADHD and related traits.</li>
<li>First study to include <b>Epigenome</b> of participants with persistent and reduced ADHD.</li>
<li>Epigenome-wide analysis was performed with the <b>Infinium® MethylationEPIC BeadChip (Illumina)</b>.</li>
<li>The <code>readEPIC()</code> function of the <code>wateRmelon</code> package was used to import <b>methylation</b> data and to compare intensities of methylated (M) and unmethylated (U) DNA.</li>
<li>The <code>Comb-p</code> Python module was used to identify differentially methylated regions (DMR) in all analyses.</li>
<li>From the assessment on DNA methylation; no global differences in DNA methylation were found between participants with persistent ADHD and healthy controls</li>
<li>Via regional analysis they identified a small DMR in the PPT2 gene (65 bp containing 4 probes) to be significantly hypomethylated in participants with persistent ADHD.</li>
<li>Hypermethylation of APOB and LPAR5 in peripheral blood might be associated with persistence of ADHD, which is probably due to genetic variation influencing DNA methylation.</li>

</ul>

</section>
</section>
<section>
<section id="slide-8">
<h2 id="8"><span class="section-number-2">8.</span> 07 Transcriptomic risk scores for attention deficit/hyperactivity disorder</h2>
<ul>
<li>Aim was to evaluate Transcription risk score (TRS) in peripheral blood mononuclear cells of individuals with ADHD and controls.</li>
<li>TWAS was performed with S-PrediXcan (python) using statistics from largest GWAS on ADHD.</li>
<li><code>56 genes</code> showed transcriptome-wide significant association with ADHD, of which <code>31</code> did not overlap with previously described GWAS loci or TWAS results by Demontis (3rd paper); might be due to differences in the tissues and methods used to construct the expression reference panel.</li>
<li>Found association between ADHD and TRSs in PBMCs constructed using TWAS results from multiple brain areas</li>
<li>Shows that individuals with ADHD carry a higher burden of TRSs than controls.</li>
<li><p>
TRSs were uncorrelated with Polygenic risk score (PRS) for ADHD.
</p>
</section>
<section>
<p>
Z-scores are plotted for the 56 genes significantly associated with ADHD. Significants in black, Z-score from negative blue, white &amp; positive red.
</p></li>

</ul>

<div id="org0365f09" class="figure">
<p><img src="file:///home/idlip/d-sync/projects/adhd/twas-56g.webp" alt="twas-56g.webp" height="960px" />
</p>
</div>

</section>
</section>
<section>
<section id="slide-9">
<h2 id="9"><span class="section-number-2">9.</span> 08 Genetics of attention deficit hyperactivity disorder</h2>
<ul>
<li>Aim was to study the ADHD's high genetic heritability of 74%.</li>
<li>The initial GWAS of ADHD did not discover any DNA variants that achieved genome-wide significance.</li>
<li>They took mixed study of twins, and previous genetic analysis.</li>
<li>Demontis describes that genes implicated by the genome-wide significant loci have relevant biological roles.
<ul>
<li><code>DUSP6</code> regulates neurotransmitter homeostasis by affecting dopamine levels</li>
<li><code>SEMA6D</code> regulates neuronal wiring during embryonic development.. so on</li>

</ul></li>
<li>Polygenic role for ADHD was confirmed by estimating polygenic risk scores in one subset of the sample.</li>
<li>There was no common DNA variants that is sufficient cause for ADHD, it remains unknown on how many variants make up for polygenic component.</li>
<li>Heritability is likely due to gene−gene interactions, gene−environment interactions or gene−environment correlations.</li>
<li>Twin estimates of heritability are less than 100%, that quite strongly suggest that environmental factors must be involved.</li>

</ul>

</section>
</section>
<section>
<section id="slide-10">
<h2 id="10"><span class="section-number-2">10.</span> 09 Discovery of the first genome-wide significant risk loci for attention-deficit/hyperactivity disorder</h2>
<ul>
<li>Aim was to study genome wide significant loci.</li>
<li>Genome-wide association meta-analysis of <b>20,183 diagnosed</b> ADHD cases and <b>35,191 controls</b> from 12 different cohorts that identifies variants surpassing genome-wide significance in 12 independent loci.</li>
<li>GWAS was conducted in each cohort (chinese, European&#x2026;) using logistic regression with the imputed additive genotype dosages.</li>
<li>No genome-wide significant <b>heterogeneity</b> was observed in the ADHD GWAS meta-analysis.</li>
<li><b>PRS</b> computed in each PGC study using iPSYCH as the training sample were consistently higher in ADHD cases as compared to controls.</li>
<li>Genome-wide significant loci on chromosomes 12 and 15 have more biological annotations supporting the co-localized genes.</li>
<li>Loci are located near genes that implicate neurodevelopmental processes that are likely to be relevant to ADHD, including <code>FOXP2</code>, <code>SORCS3</code>, and <code>DUSP6</code>.</li>
<li>The 12 significant loci are compelling, but only capture a tiny fraction of common variant risk for ADHD.</li>

</ul>
</section>
<section>
<p>
Significant genetic correlations between ADHD and other traits reveal overlap of genetic risk factors for ADHD across several groups of traits
</p>

<div id="org86b3e00" class="figure">
<p><img src="file:///home/idlip/d-sync/projects/adhd/adhd-gen-cor.jpg" alt="adhd-gen-cor.jpg" height="960px" width="900px" />
</p>
</div>

</section>
</section>
<section>
<section id="slide-11">
<h2 id="11"><span class="section-number-2">11.</span> 10 Comprehensive analysis of omics data identifies relevant gene networks for Attention-Deficit/Hyperactivity Disorder (ADHD)</h2>
<ul>
<li>Aim was to identify and characterize modules of  co-expressed genes associated with ADHD.</li>
<li>Used data from peripheral blood mononuclear cells (PBMC) of 270 ADHD cases and 279 controls &amp; identified seven ADHD-associated modules of co-expressed genes.</li>
<li>Multi-step approach was applied, first step: <b>Weighted Gene Correlation Network Analysis (WGCNA)</b> on the processed transcriptomic data.</li>
<li>The <code>WGCNA</code> identified a total of <b>27</b> modules of co-expressed genes with size ranging from <b>33 to 2191 genes</b>.</li>
<li>After multiple testing correction they identified seven ADHD-associated modules of co-expressed genes.</li>
<li>Some of them being enriched in both genetic and epigenetic signatures for ADHD and on biological pathways relevant for psychiatric disorders, such as the regulation of gene expression, epigenetic mechanisms and immune signaling.</li>
<li>Two of the ADHD-associated co-expression modules identified (<b>M1 and M7</b>), were enriched in genes involved in posttranscriptional regulation of gene expression and epigenetic modifications, two relevant pathways in the pathogenesis of ADHD.</li>
<li>Use of peripheral blood to assess gene expression signatures for the disorder highlights that the combination of multi-omics signals provides deeper and broader insights into the biological mechanisms underlying the disorder.</li>

</ul>

</section>
</section>
<section>
<section id="slide-12">
<h2 id="12"><span class="section-number-2">12.</span> 11 Epigenetics and Attention-Deficit/Hyperactivity Disorder: New Perspectives?</h2>
<ul>
<li>Aim was to understand influence of epigenetics on ADHD.</li>
<li>Epigenetic changes vary widely from one tissue to another. In the brain alone, DNA methylation can vary considerably from one region to another or from one system to another.</li>
<li>The first <b>Epigenome-Wide Association Studies (EWAS)</b> identified altered <span class="underline">methylation</span> in new genes that were not previously associated with ADHD such as <code>MYT1L</code>.</li>
<li>Studies on the association between DNA methylation and ADHD symptoms based on the candidate gene approach (mainly dopaminergic genes) are rare, compared to SNP association studies.</li>
<li>A large integrated epigenetic/genetic study of 391 children with ADHD reported an association between ADHD and DNA methylation levels at sites annotated to <code>VIPR2</code> and several novel differentially methylated positions, although none of them were genome-wide significant.</li>
<li>Ultimately, the methylation differences associated with ADHD reported so far are generally small.</li>
<li>The deregulation of <b>MicroRNAs</b> could be involved in the pathophysiology of ADHD, as synthesis of <b>miRNAs</b> is regulated by methylation of DNA.</li>
<li>Major limitation of epigenetic markers is that their variation may be secondary to the development of a trait or may be a marker of environmental exposures.</li>

</ul>

</section>
</section>
<section>
<section id="slide-13">
<h2 id="13"><span class="section-number-2">13.</span> 12 Sleep disturbances in ADHD: investigating the contribution of polygenic liability for ADHD and sleep-related phenotypes</h2>
<ul>
<li>Aim was to test whether sleep disturbances in children with ADHD are driven by polygenic liability for general population sleep phenotypes and if are they indicator of ADHD severity.</li>
<li>Compared polygenic liability for insomnia in children to their parents’ average liability.</li>
<li>Found that polygenic liability for long sleep duration was over-transmitted from parents to children with ADHD and this finding was supported in a combined analysis using an independent replication sample.</li>
<li>Could not find evidence that polygenic liability for sleep disturbances was associated with an increased risk of insomnia, poor sleep quality or hypersomnia in children with ADHD.</li>
<li>Found weak evidence that children with ADHD over-inherit polygenic liability for longer sleep duration.</li>
<li>Results also indicated that polygenic liability for sleep disturbances was not associated with an increased risk of insomnia, poor sleep quality, or hypersomnia in children with ADHD.</li>

</ul>

</section>
</section>
<section>
<section id="slide-14">
<h2 id="14"><span class="section-number-2">14.</span> 13 Treatment biomarkers for ADHD: Taking stock and moving forward</h2>
<ul>
<li>Aim was to study progress on discovery of treatment biomarkers for ADHD and their translation towards personalized treatment approaches, with focus on predictive and monitoring/response biomarkers.</li>
<li>The most promising measures for treatment prediction are <code>EEG</code>, genetic markers involved in <code>PRS</code> and <code>CNVs</code>.</li>
<li>Overall concludes that development of biomarker approaches and for the future allocation of patients to existing and novel pharmacological and non-pharmacological treatments based on their individual behavioral and neurobiological profiles, consistent with the principles of precision and personalized medicine.</li>
<li>This showcases the possible driving research on clinical and translational approach, as the existent research on ADHD biomarker to date has taken a primarily mechanistic approach.</li>
<li>Till date every biomarker research has many limitations like small samples, no validation out of sample (diversity), small variance in treatment, no longitudinal studies.</li>

</ul>

</section>
</section>
<section>
<section id="slide-15">
<h2 id="15"><span class="section-number-2">15.</span> 14 ADGRL3 genomic variation implicated in neurogenesis and ADHD links functional effects to the incretin polypeptide GIP</h2>
<ul>
<li>Aim was to analyze the potential pathogenicity effect of non-synonymous SNPs (nsSNPs) harbored in the ADGRL3 gene.</li>
<li>ADGRL3 has been shown to modulate neuronal interaction, axon guidance, and synaptic plasticity, which is highly controlled and orchestrated during CNS development.</li>
<li>Adhesion G protein-coupled receptors (aGPCRs) represents the a part in GPCR signaling pathway.</li>
<li>Utilized 1013 <code>nsSNPs</code> related to the ADGRL3 gene from three databases (977 from the NCBI, 12 from the DisGeNET, and 24 from nsSNPdpe)</li>
<li>Applied six pathogenicity prediction software tools <code>(i.e., Mutpred2, PANTHER-PSEP, PhD-SNP, Poly-Phen 2.0, PROVEAN, and SIFT)</code></li>
<li>Results indicate that marker <code>rs35106420</code> is predicted to be <b>pathogenic</b> by every analytical tool and shows the highest score in terms of conservation, genetic, and molecular effects.</li>
<li>With results they hypothesized that ADGRL3 nsSNP-related conformational changes located at the HRM domain might disrupt its interaction with GIP (gastric inhibitory polypeptide) downstream signaling.</li>
<li>Found that the <code>ADGRL3</code> <b>nsSNPS</b> were located within the hormone receptor domain (HRM) and the GPCR-Autoproteolysis Inducing (GAIN) domains.</li>

</ul>

</section>
</section>
<section>
<section id="slide-16">
<h2 id="16"><span class="section-number-2">16.</span> 15 Cry1Δ11 mutation induces ADHD-like symptoms through hyperactive dopamine D1 receptor signaling</h2>
<ul>
<li>Aim was to demonstrated that <code>Cry1Δ11</code> mice showed ADHD-like symptoms.</li>
<li><code>Cry1Δ11</code> mice were generated using <b>CRISPR/Cas9</b> technology. Donor DNA of <span class="underline">96 bp</span> in length, <span class="underline">sgRNA</span>, and in vitro–transcribed Cas9 mRNA were microinjected into E0.5 zygotes</li>
<li>Also identified that <b>hyperactive DRD1 signaling</b> may be the underlying pathological mechanism.</li>
<li>Circadian clock proteins might influence dopaminergic signaling in different ways, such as by dopamine synthesis, degradation, and signal transduction.</li>
<li><p>
In humans, the <code>CRY1</code> mutation leads to mis-splicing, skipping exon 11.
</p>
</section>
<section>
<p>
(A) Schematic representation of mouse Cry1 protein (top), genomic structure (middle), and aa sequence encoded by exon 11 of human or mouse Cry1 gene (bottom).
</p></li>

</ul>

<div id="org57ece0b" class="figure">
<p><img src="file:///home/idlip/d-sync/projects/adhd/mice-human-exon11.jpg" alt="mice-human-exon11.jpg" height="900px" />
</p>
</div>

</section>
</section>
<section>
<section id="slide-17">
<h2 id="17"><span class="section-number-2">17.</span> Bibliography</h2>
<ol>
<li><a href="https://www.tandfonline.com/doi/full/10.1080/14728214.2020.1820481">https://www.tandfonline.com/doi/full/10.1080/14728214.2020.1820481</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8295111/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8295111/</a></li>
<li><a href="https://www.nature.com/articles/s41588-022-01285-8">https://www.nature.com/articles/s41588-022-01285-8</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9250090/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9250090/</a></li>
<li><a href="https://www.medrxiv.org/content/10.1101/2023.08.02.23293439v1">https://www.medrxiv.org/content/10.1101/2023.08.02.23293439v1</a></li>
<li><a href="https://repository.ubn.ru.nl/bitstream/handle/2066/289574/289574.pdf?sequence=1">https://repository.ubn.ru.nl/bitstream/handle/2066/289574/289574.pdf?sequence=1</a></li>
<li><a href="https://www.nature.com/articles/s41380-023-02200-1">https://www.nature.com/articles/s41380-023-02200-1</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6477889/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6477889/</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6481311/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6481311/</a></li>
<li><a href="https://www.nature.com/articles/s41398-022-02182-8">https://www.nature.com/articles/s41398-022-02182-8</a></li>
<li><a href="https://www.frontiersin.org/articles/10.3389/fpsyt.2020.00579/full">https://www.frontiersin.org/articles/10.3389/fpsyt.2020.00579/full</a></li>
<li><a href="https://link.springer.com/article/10.1007/s00787-021-01931-2">https://link.springer.com/article/10.1007/s00787-021-01931-2</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9556670/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9556670/</a></li>
<li><a href="https://www.nature.com/articles/s41598-022-20343-z">https://www.nature.com/articles/s41598-022-20343-z</a></li>
<li><a href="https://insight.jci.org/articles/view/170434#SEC3">https://insight.jci.org/articles/view/170434#SEC3</a></li>

</ol>



</section>
</section>
<section>
<section id="slide-18">
<h2 id="18"><span class="section-number-2">18.</span> Glossary</h2>
<ol>
<li>Polygenic Risk Score: Score that is the effect of many genetic variants on an individual's phenotype, calculated as a weighted sum of trait-associated alleles</li>
<li>PBMD: peripheral blood cell with round nucleus</li>

</ol>

<style type="text/css">
.reveal section p {
    display: inline-block;
    font-size: 0.5em;
    line-height: 1.2em;
    vertical-align: top;
}

.reveal .title {
    font-size: 2.2em;
}

.reveal table {
  display: block;
  font-size: 0.5em;
  overflow: auto;
  max-height: 600px;
}

div.legalese {
  font-size: 1.4vh;
  position: fixed;
  left: 3px;
  bottom: 3px;
  z-index: 50;
}
</style>
</section>
</section>
</div>
</div>
<script src="../revjs/dist/reveal.js"></script>
<script src="../revjs/plugin/markdown/markdown.js"></script>
<script src="../revjs/plugin/notes/notes.js"></script>
<script src="../revjs/plugin/search/search.js"></script>
<script src="../revjs/plugin/zoom/zoom.js"></script>
<script>
// Full list of configuration options available here:
// https://github.com/hakimel/reveal.js#configuration
Reveal.initialize({

controls: true,
progress: false,
history: false,
center: true,
slideNumber: 'c',
rollingLinks: false,
keyboard: true,
mouseWheel: false,
fragmentInURL: false,
hashOneBasedIndex: false,
pdfSeparateFragments: true,
overview: true,
width: 1920,
height: 1080,
margin: 0.02,
minScale: 0.20,
maxScale: 1.00,

transition: 'convex',
transitionSpeed: 'default',
navigationMode: 'linear', controls: true,

// Plugins with reveal.js 4.x
plugins: [ RevealMarkdown, RevealNotes, RevealSearch, RevealZoom ],

// Optional libraries used to extend reveal.js
dependencies: [
]

});
</script>
</body>
</html>
