<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<title>ADHD: The Focus</title>
<meta name="author" content="Dilip G"/>
<style type="text/css">
.underline { text-decoration: underline; }
</style>
<link rel="stylesheet" href="revjs/dist/reveal.css"/>

<link rel="stylesheet" href="revjs/dist/theme/dracula.css" id="theme"/>
</head>
<body>
<div class="reveal">
<div class="slides">
<section id="sec-title-slide">
<h1 class="title">ADHD: The Focus</h1> <br> <br> <h2 class="subtitle"></h2> <h2 class="author">Dilip G</h2> <br> <br> <h4 class="misc"></h4> <h3 class="misc"></h3>
</section>
<section id="table-of-contents-section">
<nav id="table-of-contents" role="doc-toc">
<h2 class="org-toc">Table of Contents</h2>
<div id="text-table-of-contents" role="doc-toc">
<ul>
<li><a href="#/slide-orgf557e0b">Introduction</a></li>
<li><a href="#/slide-orgd4aafb9">Literature Review</a></li>
<li><a href="#/slide-org58681ac">R/Python Codes</a></li>
<li><a href="#/slide-orgb5f221d">Bibliography</a></li>
<li><a href="#/slide-org9b8bf38">Thank you for your Attention!</a></li>
<li><a href="#/slide-org54fbdd5">Glossary</a></li>
</ul>
</div>
</nav>
</section>

<section>
<section id="slide-orgf557e0b">
<h2 id="orgf557e0b">Introduction</h2>
</section>
<section>
<ul>
<li class="fragment appear">Attention Deficit/Hyperactivity Disorder - <b>Neuro-developmental</b> disorder, chronic condition characterized by <span class="underline">hyperactivity-impulsivity &amp; inattention</span>.</li>
<li class="fragment appear"><b>Impairment</b> in multiple domains of daily living.</li>
<li class="fragment appear">Nosological system (DSM &amp; ICD) set the criteria to classify disease or disorder.</li>
<li class="fragment appear">Prevalence of <code>5.8%</code> in children &amp; adolescents, 2.5% of adults.</li>
<li class="fragment appear">It can <span class="underline">comorbid</span> with other conditions.</li>
<li class="fragment appear">There is no cure as such, but medication and treatment (CBT) are viable.</li>
<li class="fragment appear">Treatments can be stimulant based or non-stimulant, but stimulants are said to be more effective.</li>
<li class="fragment appear">Usual treatments target <b>dopaminergic &amp; noradrenegic</b> system.</li>

</ul>

</section>
</section>
<section>
<section id="slide-orgd4aafb9">
<h2 id="orgd4aafb9">Literature Review</h2>
<div class="outline-text-2" id="text-orgd4aafb9">
</div>
</section>
<section id="slide-org88ae8ef">
<h3 id="org88ae8ef">Genetics of attention deficit hyperactivity disorder</h3>
<p>
<b><i>Faraone SV et al,</i></b>  Jun 11, 2018
</p>
<ul>
<li>Aim was to study the ADHD&rsquo;s high <b>genetic heritability of 74%</b>.</li>
<li>The initial <b>GWAS</b> of ADHD did not discover any DNA variants that achieved genome-wide significance.</li>
<li>They took mixed study of twins, and previous genetic analysis.</li>
<li><i>Demontis</i> describes that genes implicated by the genome-wide significant loci have relevant biological roles.
<ul>
<li><code>DUSP6</code> regulates neurotransmitter homeostasis by affecting dopamine levels</li>
<li><code>SEMA6D</code> regulates neuronal wiring during embryonic development.. so on</li>

</ul></li>
<li><b>Polygenic</b> role for ADHD was confirmed by estimating polygenic risk scores in one subset of the sample.</li>
<li>There was no common DNA variants that is sufficient cause for ADHD, it remains unknown on how many variants make up for polygenic component.</li>
<li><b>Heritability</b> is likely due to <span class="underline">gene−gene interactions, gene−environment interactions or gene−environment correlations</span>.</li>
<li>Twin estimates of heritability are less than 100%, that quite strongly suggest that environmental factors must be involved.</li>

</ul>

</section>
<section id="slide-org4d3668f">
<h3 id="org4d3668f">Genome-wide DNA methylation in association with ADHD, and in association with impulsive and callous traits</h3>
<p>
<b><i>M. Meijer et al,</i></b>  Jan 31, 2020
</p>
<ul>
<li>To study DNA <b><span class="underline">methylation</span></b> in persistent ADHD and related traits.</li>
<li>First study to include <b>Epigenome</b> of participants with persistent and reduced ADHD.</li>
<li>Epigenome-wide analysis was performed with the <b>Infinium® MethylationEPIC BeadChip (Illumina)</b>.</li>
<li>The <code>readEPIC()</code> function of the <code>wateRmelon</code> package was used to import <b>methylation</b> data and to compare intensities of methylated (M) and unmethylated (U) DNA.</li>
<li>The <code>Comb-p</code> Python module was used to identify differentially methylated regions (DMR) in all analyses.</li>
<li>From the assessment on DNA methylation; no global differences in DNA methylation were found between participants with persistent ADHD and healthy controls</li>
<li>Via regional analysis they identified a small DMR in the <code>PPT2</code> gene (65 bp containing 4 probes) to be significantly hypomethylated in participants with persistent ADHD.</li>
<li><b>Hypermethylation</b> of <code>APOB</code> and <code>LPAR5</code> in peripheral blood might be associated with persistence of ADHD, which is probably due to genetic variation influencing DNA methylation.</li>

</ul>

</section>
<section id="slide-org42854db">
<h3 id="org42854db">Epigenetics and Attention-Deficit/Hyperactivity Disorder: New Perspectives?</h3>
<p>
<b><i>Bojan Mirkovic et al,</i></b>  June 17, 2020
</p>
<ul>
<li>Aim was to understand influence of <b>epigenetics</b> on ADHD.</li>
<li>Epigenetic changes vary widely from one tissue to another. In the brain alone, DNA methylation can vary considerably from one region to another or from one system to another.</li>
<li>The first <b>Epigenome-Wide Association Studies (EWAS)</b> identified altered <span class="underline">methylation</span> in new genes that were not previously associated with ADHD such as <code>MYT1L</code>.</li>
<li>Studies on the association between DNA methylation and ADHD symptoms based on the candidate gene approach (mainly dopaminergic genes) are rare, compared to SNP association studies.</li>
<li>A large integrated epigenetic/genetic study of <span class="underline">391 children</span> with ADHD reported an association between ADHD and DNA methylation levels at sites annotated to <code>VIPR2</code> and other positions, although none of them were genome-wide significant.</li>
<li>Ultimately, the methylation differences associated with ADHD reported so far are generally small.</li>
<li>The deregulation of <b>MicroRNAs</b> could be involved in the pathophysiology of ADHD, as synthesis of <b>miRNAs</b> is regulated by methylation of DNA.</li>

</ul>

</section>
<section id="slide-org7f04d98">
<h3 id="org7f04d98">Discovery of the first genome-wide significant risk loci for attention-deficit/hyperactivity disorder</h3>
<p>
<b><i>Demontis D et al,</i></b>  Aug 12, 2020
</p>
<ul>
<li>Aim was to study genome wide significant loci.</li>
<li>Genome-wide association meta-analysis of <b>20,183 diagnosed</b> ADHD cases and <b>35,191 controls</b> from 12 different <span class="underline">cohorts</span> that identifies variants surpassing genome-wide significance in 12 independent loci.</li>
<li>GWAS was conducted in each cohort (chinese, European&#x2026;) using logistic regression with the imputed additive genotype dosages.</li>
<li>No genome-wide significant <b>heterogeneity</b> was observed in the ADHD GWAS meta-analysis.</li>
<li><b>PRS</b> computed in each <b>PGC</b> study using <b>iPSYCH</b> as the training sample were consistently higher in ADHD cases as compared to controls.</li>
<li>Genome-wide significant loci on chromosomes 12 and 15 have more biological annotations supporting the co-localized genes.</li>
<li>Loci are located near genes that implicate neurodevelopmental processes that are likely to be relevant to ADHD, including <code>FOXP2</code>, <code>SORCS3</code>, and <code>DUSP6</code>.</li>
<li>The <b>12 significant loci</b> are compelling, but only capture a tiny fraction of common variant risk for ADHD.</li>

</ul>
</section>
<section>
<p>
Significant genetic correlations between ADHD and other traits reveal overlap of genetic risk factors for ADHD across several groups of traits
</p>

<figure id="org7a830ee">
<img src="images/adhd-gen-cor.jpg" alt="adhd-gen-cor.jpg" height="960px" width="900px">

</figure>

</section>
<section id="slide-org4d2c42d">
<h3 id="org4d2c42d">Emerging drugs for the treatment of attention-deficit hyperactivity disorder (ADHD)</h3>
<p>
<b><i>Marco Pozzi et al</i></b>  Sept 25, 2020
</p>
<ul>
<li>Aim was to study pharmacological treatment of ADHD made during 2010-2020.</li>
<li>During studies there were dropouts mainly due to side effects.</li>
<li>About 20% of treated patients could not tolerate their therapies or experience insufficient efficacy.</li>
<li><code>Amantadine</code> to enhance cognitive functions and <code>tipepidine</code> for hyperactivity/impulsivity. Stand-alone emerging treatments for ADHD include <code>viloxazine</code> and <code>dasotraline</code> (pediatric)</li>
<li>Monoaminergic (neurotransmitters) reuptake inhibitors have a great potential to improve ADHD symptoms.</li>

</ul>

</section>
<section id="slide-org0a90c02">
<h3 id="org0a90c02">ADHD at the workplace: ADHD symptoms, diagnostic status, and work-related functioning</h3>
<p>
<b><i>Anselm B. M. Fuermaier et al</i></b> Feb 2, 2021
</p>
<ul>
<li>Aim was to examine the nature of work-related problems and impairments of adults with ADHD, and neuropsychological test performance.</li>
<li>Sample size of 1231 individuals from workplace and other 134 adults diagnosed from ADHD clinic.</li>
<li>It was questionnaire &amp; Neuropsychological performance tests based evaluated using <code>Conners’ Adult ADHD Rating Scale (CAARS)</code>: 66 items, scaled on four-points and followed by <code>Weiss Functional Impairment Rating Scale (WFIRS)</code>: 70 items, scored on four-point scale.</li>
<li>Statistically Analysis concludes that work-related issues are common in individuals diagnosed with ADHD but also in individuals with symptoms of ADHD.</li>
<li>Individuals with ADHD experienced problems at work in particular in not meeting their own standards and perceived potential.</li>
<li>Although some neuropsychological tests did not predict work-related problems.</li>

</ul>

</section>
<section id="slide-org846ed0d">
<h3 id="org846ed0d">Deep learning model reveals potential risk genes for ADHD, especially Ephrin receptor gene EPHA5</h3>
<p>
<b><i>Lu Liu et al.</i></b>  Jun 09, 2021; published in <i>Briefings in Bioinformatics, Volume 22</i>
</p>
<ul>
<li>Aim was to propose a <b>convolutional neural network-based deep learning model</b> for the classification of ADHD.</li>
<li>Results indicated that the deep learning model could capture the relationship between <code>SNPs</code> with insignificant <b>P-values</b>, while GWAS failed (needs lower P-value).</li>
<li>Identified a potential risk gene for ADHD, <code>EPHA5</code> with a variant of <code>rs4860671</code>.</li>
<li>They applied <b>saliency map analysis</b> to the deep learning model and found potential ADHD-associated SNPs and genes.</li>
<li>Deep learning model achieved high accuracy of <b>0.9018</b>, <b>AUC of 0.9570</b> on their testing dataset.</li>
<li>This is a first deep learning method for the classification of ADHD with SNPs data.</li>

</ul>

</section>
<section id="slide-org923b2cb">
<h3 id="org923b2cb">ADGRL3 genomic variation implicated in neurogenesis and ADHD links functional effects to the incretin polypeptide GIP</h3>
<p>
<b><i>Vidal, O.M et al,</i></b>  Feb 18, 2022
</p>
<ul>
<li>Aim was to analyze the potential pathogenicity effect of non-synonymous SNPs (nsSNPs) harbored in the ADGRL3 gene.</li>
<li><code>ADGRL3</code> has been shown to modulate neuronal interaction, axon guidance, and synaptic plasticity, which is highly controlled and orchestrated during CNS development.</li>
<li>Utilized 1013 <code>nsSNPs</code> related to the <code>ADGRL3</code> gene from three databases (977 from the NCBI, 12 from the DisGeNET, and 24 from nsSNPdpe)</li>
<li>Applied six pathogenicity prediction software tools <code>(i.e., Mutpred2, PANTHER-PSEP, PhD-SNP, Poly-Phen 2.0, PROVEAN, and SIFT)</code></li>
<li>Results indicate that marker <code>rs35106420</code> is predicted to be <b>pathogenic</b> by every analytical tool and shows the highest score in terms of conservation, genetic, and molecular effects.</li>
<li>With results they hypothesized that ADGRL3 nsSNP-related conformational changes located at the HRM domain might disrupt its interaction with GIP (gastric inhibitory polypeptide) downstream signaling.</li>
<li>Found that the <code>ADGRL3</code> <b>nsSNPS</b> were located within the hormone receptor domain (HRM) and the GPCR-Autoproteolysis Inducing (GAIN) domains.</li>

</ul>

</section>
<section id="slide-org1e4b66f">
<h3 id="org1e4b66f">Effects of physical exercise on children with attention deficit hyperactivity disorder</h3>
<p>
<b><i>Yuan-Shuo Chan et al</i></b>,  Apr 2022
</p>
<ul>
<li>To Examine effects of physical exercise in children with ADHD.</li>
<li>Aerobic moderate–high intensity exercise can promote <b>BNDF synthesis</b>, enhance neuroplasticity, and decrease the risk of nervous system disorders.</li>
<li>Intervention of single-bout of exercise offers immediate and persistent benefits in cognitive function, and long-term exercise training can effectively improve overall fitness and exert long-lasting effects on cognitive function.</li>
<li>At early stage due to impulsiveness, lack of motivation, impatience, and persistence of children with ADHD, interval training (HIIT) is the most recommended form of exercise.</li>
<li>Emphasized on perceptual motor exercises combined with cognitive tasks to improve cognitive function of children with ADHD.</li>

</ul>

</section>
<section id="slide-orgc5650dd">
<h3 id="orgc5650dd">Comprehensive analysis of omics data identifies relevant gene networks for Attention-Deficit/Hyperactivity Disorder (ADHD)</h3>
<p>
<b><i>Cabana-Domínguez et al,</i></b>  Sept 24, 2022
</p>
<ul>
<li>Aim was to identify and characterize modules of  co-expressed genes associated with ADHD.</li>
<li>Used data from <b>peripheral blood mononuclear cells (PBMC)</b> of 270 ADHD cases and 279 controls &amp; identified seven ADHD-associated modules of co-expressed genes.</li>
<li>Multi-step approach was applied, first step: <b>Weighted Gene Correlation Network Analysis (WGCNA)</b> on the processed transcriptomic data.</li>
<li>The <code>WGCNA</code> identified a total of <b>27</b> modules of co-expressed genes with size ranging from <b>33 to 2191 genes</b>.</li>
<li>After multiple testing correction they identified <b><span class="underline">seven</span></b> ADHD-associated modules of co-expressed genes.</li>
<li>Use of peripheral blood to assess gene expression signatures for the disorder highlights that the combination of multi-omics signals provides deeper and broader insights into the biological mechanisms underlying the disorder.</li>

</ul>

</section>
<section id="slide-org0392be5">
<h3 id="org0392be5">Treatment biomarkers for ADHD: Taking stock and moving forward</h3>
<p>
<b><i>Michelini G et al,</i></b>  Oct 12, 2022
</p>
<ul>
<li>Aim was to study progress on discovery of treatment biomarkers for ADHD and their translation towards personalized treatment approaches, with focus on predictive and monitoring/response biomarkers.</li>
<li>The most promising measures for treatment prediction are <code>EEG</code> (electroencephalogram), genetic markers involved in <code>PRS</code> and <code>CNVs</code>.</li>
<li>Overall concludes that development of biomarker approaches is consistent with the principles of precision and personalized medicine.</li>
<li>This showcases the possible driving research on <b>clinical</b> and <b>translational</b> approach, as the existent research on ADHD biomarker to date has taken a primarily <span class="underline">mechanistic</span> approach.</li>
<li>Till date every biomarker research has many limitations like small samples, no validation out of sample (diversity), small variance in treatment, no longitudinal studies.</li>

</ul>

</section>
<section id="slide-org854e1e3">
<h3 id="org854e1e3">Sleep disturbances in ADHD: investigating the contribution of polygenic liability for ADHD and sleep-related phenotypes</h3>
<p>
<b><i>Lewis, K.J.S., Martin J et al,</i></b>  Jan 07, 2023
</p>
<ul>
<li>Aim was to test whether sleep disturbances in children with ADHD are driven by polygenic liability for general population sleep phenotypes and if are they indicator of ADHD severity.</li>
<li>Compared polygenic liability for insomnia in children to their parents’ average liability.</li>
<li>Found that polygenic liability for long sleep duration was over-transmitted from parents to children with ADHD and this finding was supported in a combined analysis using an independent replication sample.</li>
<li>Could not find evidence that polygenic liability for sleep disturbances was associated with an increased risk of insomnia, poor sleep quality or hypersomnia in children with ADHD.</li>
<li>Found weak evidence that children with ADHD over-inherit polygenic liability for longer sleep duration.</li>
<li>Results also indicated that polygenic liability for sleep disturbances was not associated with an increased risk of insomnia, poor sleep quality, or hypersomnia in children with ADHD.</li>

</ul>

</section>
<section id="slide-orga3634fb">
<h3 id="orga3634fb">Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains</h3>
<p>
<b><i>Demontis, D et al</i></b>, Jan 26, 2023
</p>
<ul>
<li>GWAS of <b>38,691 Individuals &amp; 186,843 controls</b> with ADHD (largest uptake).</li>
<li>Data source from <code>iPSYCH</code> (25,895 cases;  37,148 controls), <code>deCODE genetics</code> (8,281 cases; 137,993 controls) and previously published  data from 10 ADHD cohorts with European ancestry collected by the <code>PGC</code> (4,515 cases; 11,702  controls),</li>
<li>Performed transcriptome-wide  association study (TWAS) of the genetically regulated gene expression using <code>EpiXcan</code> to identify risk genes on 924 samples.</li>
<li>TWAS identified 23 distinct genes with different gene expression level in ADHD compared to Control.</li>
<li>Identified <b>27 genome-wide significant loci</b>, double compared to previous analysis (<b>ADHD2019</b>).</li>
<li>Three of the strongest associated loci (<b>p &lt; 5x10^-14 )</b> were on <b>chromosome 1, 5 and 11</b>.</li>
<li>Observed that ADHD risk genes are significant in frontal cortex of brain in cells like inhibitory, excitatory and dopaminergic neuron.</li>
<li>Summarize that ADHD is polygenic and also is influenced by other psychiatric disorders.</li>

</ul>

</section>
<section id="slide-org51baead">
<h3 id="org51baead">Transcriptomic risk scores for attention deficit/hyperactivity disorder</h3>
<p>
<b><i>Judit Cabana-Domínguez et al,</i></b>   Aug 03, 2023
</p>
<ul>
<li>Aim was to evaluate <b>Transcription risk score (TRS)</b> in peripheral blood mononuclear cells of individuals with ADHD and controls.</li>
<li>TWAS was performed with <code>S-PrediXcan</code> (python) using statistics from largest GWAS on ADHD.</li>
<li><code>56 genes</code> showed transcriptome-wide significant association with ADHD, of which <code>31</code> did not overlap with previously described GWAS loci or TWAS results by Demontis (previous paper); might be due to differences in the tissues and methods used to construct the expression reference panel.</li>
<li>Found association between ADHD and <b>TRSs</b> in <b>PBMCs</b> constructed using TWAS results from multiple brain areas.</li>
<li>Shows that individuals with ADHD carry a higher burden of TRSs than controls.</li>
<li><p>
<b>TRSs</b> were uncorrelated with Polygenic risk score (<b>PRS</b>) for ADHD.
</p>
</section>
<section>
<p>
Z-scores are plotted for the 56 genes significantly associated with ADHD. Significants in black, Z-score from negative blue, white &amp; positive red.
</p></li>

</ul>

<figure id="org358ba66">
<img src="images/twas-56g.webp" alt="twas-56g.webp" height="960px">

</figure>

</section>
<section id="slide-orga1f5a16">
<h3 id="orga1f5a16">Examining Differences in the Genetic and Functional Architecture of ADHD Diagnosed in Childhood and Adulthood</h3>
<p>
<b><i>Sophie Breunig et al</i></b>,  Aug 07, 2023
</p>
<ul>
<li>Aim was to investigate whether individuals who are diagnosed as children differ from those who are diagnosed in adulthood with respect to shared and unique architecture at the genome-wide, functional, and gene expression level of analysis.</li>
<li>Used <b>Genomic SEM (structural equation modeling)</b> to investigate genetic correlations of GWAS sample taken from <b>iPSYCH</b>.</li>
<li>Then applied Stratified Genomic SEM and transcriptome-wide SEM (T-SEM) to identify functional annotations and patterns of gene expression associated with genetic risk sharing or divergence across the ADHD subgroups.</li>
<li>At the functional and gene expression levels, Stratified Genomic SEM and T-SEM analyses revealed three annotations and <b>22 genes</b>, respectively, that were significantly associated with genetic risk sharing across the subtypes.</li>
<li>Adulthood diagnosed ADHD exhibited a significantly larger negative <b>r_g</b> (genetic correlation) with educational attainment.</li>
<li>The genetic correlation (<b>r_g</b>) across childhood and adulthood diagnosed ADHD was 0.76 (SEr = 0.06).</li>

</ul>

</section>
<section id="slide-orge5763cc">
<h4 id="orge5763cc">T-SEM Pinpoints Genes Associated with General ADHD</h4>
<ul>
<li>Univariate TWAS  reports two significant gene expression associated with adulthood and 19 genes associated with childhood ADHD.</li>
<li>Miami plot for gene expression hits for the combined factor of childhood and adulthood diagnosed ADHD.</li>

</ul>
<p>
The upper and lower blue lines represent the Bonferroni corrected significance threshold. Genes surpassing the upper and lower cutoff respectively are upwardly and downwardly regulated respectively in the ADHD factor. The most significant gene across tissue types are labeled and colored as red dots.
<img src="images/gene-exp-plot.jpg" alt="gene-exp-plot.jpg">
</p>

</section>
<section id="slide-org44c37c9">
<h3 id="org44c37c9">Cry1Δ11 mutation induces ADHD-like symptoms through hyperactive dopamine D1 receptor signaling</h3>
<p>
<b><i>Dengfeng Liu et al</i></b>  Aug 22, 2023
</p>
<ul>
<li>Aim was to demonstrate that <code>Cry1Δ11</code> mice showed ADHD-like symptoms.</li>
<li><code>Cry1Δ11</code> mice were generated using <b>CRISPR/Cas9</b> technology. Donor DNA of <span class="underline">96 bp</span> in length, <span class="underline">sgRNA</span>, and in vitro–transcribed Cas9 mRNA were microinjected into E0.5 zygotes</li>
<li>Also identified that <b>hyperactive DRD1 signaling</b> may be the underlying pathological mechanism.</li>
<li>Circadian clock proteins might influence dopaminergic signaling in different ways, such as by dopamine synthesis, degradation, and signal transduction.</li>
<li>In humans, the <code>CRY1</code> mutation leads to mis-splicing, skipping exon 11.</li>

</ul>


<figure id="org78bc270">
<img src="images/mice-human-exon11.jpg" alt="mice-human-exon11.jpg" height="260px" align="left">

<figcaption><span class="figure-number">Figure 1: </span>(A) Schematic representation of mouse Cry1 protein (top), genomic structure (middle), and aa sequence encoded by exon 11 of human or mouse Cry1 gene (bottom).</figcaption>
</figure>

</section>
</section>
<section>
<section id="slide-org58681ac">
<h2 id="org58681ac">R/Python Codes</h2>
<div class="outline-text-2" id="text-org58681ac">
</div>
</section>
<section id="slide-org65e05dd">
<h3 id="org65e05dd">ggplot on drug molecules &amp; patients it was addressed</h3>
<div class="org-src-container">

<pre class="src src-R" id="ggplot-mol"><span class="org-comment-delimiter">## </span><span class="org-comment">install it by running: install.packages("ggplot2")</span>

<span class="org-comment-delimiter">## </span><span class="org-comment">call library</span>
<span class="org-ess-modifiers">library</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"ggplot2"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">import adhd data from csv</span>
adhd <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">read.csv</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"exp-treatment-10y.csv"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">making plot with Molecule on x-axis</span>
<span class="org-comment-delimiter">## </span><span class="org-comment">no of pateints in y-axis</span>
myplot <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">ggplot</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>adhd, <span class="org-ess-function-call">aes</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">(</span></span>x <span class="org-ess-operator">=</span> Molecule, y <span class="org-ess-operator">=</span> patients, color <span class="org-ess-operator">=</span> Molecule<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">)</span></span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span> <span class="org-ess-operator">+</span> <span class="org-ess-function-call">geom_point</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>size <span class="org-ess-operator">=</span> <span class="org-ess-numbers">10</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">rotate the name to not overlap and increase font size</span>
myplot <span class="org-ess-operator">+</span> <span class="org-ess-function-call">theme</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>axis.text.x <span class="org-ess-operator">=</span> <span class="org-ess-function-call">element_text</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">(</span></span>angle <span class="org-ess-operator">=</span> <span class="org-ess-numbers">45</span>, vjust <span class="org-ess-operator">=</span> <span class="org-ess-numbers">1</span>, hjust <span class="org-ess-operator">=</span> <span class="org-ess-numbers">1</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">)</span></span>, text <span class="org-ess-operator">=</span> <span class="org-ess-function-call">element_text</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">(</span></span>size <span class="org-ess-operator">=</span> <span class="org-ess-numbers">23</span>, face <span class="org-ess-operator">=</span> <span class="org-string">"bold"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">)</span></span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">save the plot to png file with proper aspect ratio</span>
<span class="org-ess-function-call">ggsave</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"pat-mol.png"</span>, width <span class="org-ess-operator">=</span> <span class="org-ess-numbers">1920</span>, height <span class="org-ess-operator">=</span> <span class="org-ess-numbers">1080</span>,  units <span class="org-ess-operator">=</span> <span class="org-string">"px"</span>, dpi <span class="org-ess-operator">=</span> <span class="org-ess-numbers">100</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>
</pre>
</div>

</section>
<section>

<figure id="org1384d16">
<img src="images/pat-mol.png" alt="pat-mol.png">

</figure>

</section>
<section id="slide-orga1de9e1">
<h3 id="orga1de9e1">matplot on drug molecule &amp; patients count</h3>
<div class="org-src-container">

<pre class="src src-python" id="matplot-mol"><span class="org-doc">"""</span>
<span class="org-doc">plot in python using matplotlib library, and pandas for dataframes</span>
<span class="org-doc">"""</span>

<span class="org-comment-delimiter"># </span><span class="org-comment">get a module from matplotlib library as plt (name)</span>
<span class="org-keyword">import</span> matplotlib.pyplot <span class="org-keyword">as</span> plt
<span class="org-comment-delimiter"># </span><span class="org-comment">import read_csv function from pandas</span>
<span class="org-keyword">from</span> pandas <span class="org-keyword">import</span> read_csv <span class="org-keyword">as</span> readcsv

<span class="org-comment-delimiter"># </span><span class="org-comment">capture from csv file</span>
<span class="org-variable-name">adhd</span> <span class="org-operator">=</span> readcsv<span class="org-rainbow-delimiters-depth-1">(</span><span class="org-string">'exp-treatment-10y.csv'</span><span class="org-rainbow-delimiters-depth-1">)</span>

<span class="org-comment-delimiter"># </span><span class="org-comment">assign the list to x and y axis</span>
<span class="org-variable-name">x</span> <span class="org-operator">=</span> adhd<span class="org-rainbow-delimiters-depth-1">[</span><span class="org-string">'Molecule'</span><span class="org-rainbow-delimiters-depth-1">]</span>
<span class="org-variable-name">y</span> <span class="org-operator">=</span> adhd<span class="org-rainbow-delimiters-depth-1">[</span><span class="org-string">'patients'</span><span class="org-rainbow-delimiters-depth-1">]</span>

<span class="org-comment-delimiter"># </span><span class="org-comment">increase font size</span>
plt.rcParams.update<span class="org-rainbow-delimiters-depth-1">(</span><span class="org-rainbow-delimiters-depth-2">{</span><span class="org-string">'font.size'</span>: 11, <span class="org-string">'font.weight'</span>: <span class="org-string">'bold'</span><span class="org-rainbow-delimiters-depth-2">}</span><span class="org-rainbow-delimiters-depth-1">)</span>

<span class="org-comment-delimiter"># </span><span class="org-comment">make a plot</span>
plt.plot<span class="org-rainbow-delimiters-depth-1">(</span>x, y<span class="org-rainbow-delimiters-depth-1">)</span>

<span class="org-comment-delimiter"># </span><span class="org-comment">rotate the x axis labels to it does not overlap</span>
plt.xticks<span class="org-rainbow-delimiters-depth-1">(</span>rotation<span class="org-operator">=</span>45<span class="org-rainbow-delimiters-depth-1">)</span>

plt.title<span class="org-rainbow-delimiters-depth-1">(</span><span class="org-string">"Number of Patients to each Drug Molecule"</span><span class="org-rainbow-delimiters-depth-1">)</span>

plt.xlabel<span class="org-rainbow-delimiters-depth-1">(</span><span class="org-string">'Drug Molecules'</span><span class="org-rainbow-delimiters-depth-1">)</span>
plt.ylabel<span class="org-rainbow-delimiters-depth-1">(</span><span class="org-string">'Patients count'</span><span class="org-rainbow-delimiters-depth-1">)</span>

<span class="org-comment-delimiter"># </span><span class="org-comment">saving the final plot</span>
plt.savefig<span class="org-rainbow-delimiters-depth-1">(</span><span class="org-string">'pat-mol2.png'</span><span class="org-rainbow-delimiters-depth-1">)</span>

<span class="org-comment-delimiter"># </span><span class="org-comment">shows the final plot when run</span>
plt.show<span class="org-rainbow-delimiters-depth-1">()</span>
</pre>
</div>

</section>
<section>

<figure id="org204aec8">
<img src="images/pat-mol2.png" alt="pat-mol2.png">

</figure>

</section>
<section id="slide-orgb1df20a">
<h3 id="orgb1df20a">calculate nucleotide content</h3>
<div class="org-src-container">

<pre class="src src-R" id="nucleo-cont"><span class="org-comment-delimiter">## </span><span class="org-comment">call biostring library</span>
<span class="org-ess-modifiers">library</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"Biostrings"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">get fasta file and read the sequence strings</span>
gene <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">readDNAStringSet</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"../fasta/dusp6-gene.fna"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-ess-function-call">print</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>gene<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">translate gene content to amino acids units</span>
protein <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">translate</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>gene<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>
<span class="org-ess-function-call">print</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>protein<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">calculate the number of bases in the sequence</span>
base_count <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">letterFrequency</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>gene, letters <span class="org-ess-operator">=</span> <span class="org-string">"ACGT"</span>, OR <span class="org-ess-operator">=</span> <span class="org-ess-numbers">0</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>
<span class="org-ess-function-call">print</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>base_count<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">make a vector to plot</span>
counts <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">as.vector</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-ess-function-call">letterFrequency</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">(</span></span>gene<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-3">[</span></span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-4">[</span></span><span class="org-ess-numbers">1</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-4">]</span></span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-3">]</span></span>, letters <span class="org-ess-operator">=</span> <span class="org-string">"ACGT"</span>, OR <span class="org-ess-operator">=</span> <span class="org-ess-numbers">0</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">)</span></span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>
bases <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">c</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"A"</span>, <span class="org-string">"C"</span>, <span class="org-string">"G"</span>, <span class="org-string">"T"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-ess-modifiers">library</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"ggplot2"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-comment-delimiter">## </span><span class="org-comment">plot</span>
acgt <span class="org-ess-assignment">&lt;-</span> <span class="org-ess-function-call">ggplot</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>mapping <span class="org-ess-operator">=</span> <span class="org-ess-function-call">aes</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">(</span></span>x <span class="org-ess-operator">=</span> bases, y <span class="org-ess-operator">=</span> counts, color <span class="org-ess-operator">=</span> <span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-3">(</span></span>bases<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-3">)</span></span>, fill <span class="org-ess-operator">=</span> bases<span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">)</span></span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span> <span class="org-ess-operator">+</span>
    <span class="org-ess-function-call">geom_point</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>size <span class="org-ess-operator">=</span> <span class="org-ess-numbers">10</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span> <span class="org-ess-operator">+</span> <span class="org-ess-function-call">geom_bar</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>stat <span class="org-ess-operator">=</span> <span class="org-string">"identity"</span>, width <span class="org-ess-operator">=</span> <span class="org-ess-numbers">0.5</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span> <span class="org-ess-operator">+</span> <span class="org-ess-function-call">theme</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span>text <span class="org-ess-operator">=</span> <span class="org-ess-function-call">element_text</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">(</span></span>size <span class="org-ess-operator">=</span> <span class="org-ess-numbers">23</span>, face <span class="org-ess-operator">=</span> <span class="org-string">"bold"</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-2">)</span></span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>

<span class="org-ess-function-call">ggsave</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">(</span></span><span class="org-string">"base-count.png"</span>, width <span class="org-ess-operator">=</span> <span class="org-ess-numbers">1920</span>, height <span class="org-ess-operator">=</span> <span class="org-ess-numbers">1080</span>,  units <span class="org-ess-operator">=</span> <span class="org-string">"px"</span>, dpi <span class="org-ess-operator">=</span> <span class="org-ess-numbers">100</span><span class="org-ess-paren"><span class="org-rainbow-delimiters-depth-1">)</span></span>
</pre>
</div>

</section>
<section>

<figure id="org5bb45a9">
<img src="images/base-count.png" alt="base-count.png">

</figure>

</section>
</section>
<section>
<section id="slide-orgb5f221d">
<h2 id="orgb5f221d">Bibliography</h2>
</section>
<section>
<ol>
<li>Faraone SV, Larsson H. Genetics of attention deficit hyperactivity disorder. Mol Psychiatry. 2019 Apr;24(4):562-575. doi: 10.1038/s41380-018-0070-0. Epub 2018 Jun 11. PMID: 29892054; PMCID: PMC6477889. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6477889/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6477889/</a></li>

<li>M. Meijer, M. Klein, et al, Genome-wide DNA methylati on in associati on with persistent att enti on-defi cit/hyperacti vity disorder, and in associati on with impulsive and callous traits. Fronti ers in Geneti cs, 2020, DOI: 10.3389/fgene.2020.00016. <a href="https://www.frontiersin.org/articles/10.3389/fgene.2020.00016/full">https://www.frontiersin.org/articles/10.3389/fgene.2020.00016/full</a></li>

<li>Mirkovic Bojan, Chagraoui Abdeslam, Gerardin Priscille, Cohen David. Epigenetics and Attention-Deficit/Hyperactivity Disorder: New Perspectives?. Frontiers in Psychiatry 11 , 2020 DOI=10.3389/fpsyt.2020.00579 <a href="https://www.frontiersin.org/articles/10.3389/fpsyt.2020.00579/full">https://www.frontiersin.org/articles/10.3389/fpsyt.2020.00579/full</a></li>

<li>Demontis D. et al, Discovery of the first genome-wide significant risk loci for attention deficit/hyperactivity disorder. Nat Genet. 2019 Jan;51(1):63-75. doi: 10.1038/s41588-018-0269-7. Epub 2018 Nov 26. PMID: 30478444; PMCID: PMC6481311. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6481311/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6481311/</a></li>

<li>Marco Pozzi et al, Emerging drugs for the treatment of attention-deficit hyperactivity disorder (ADHD), Expert Opinion on Emerging Drugs, 25:4, 395-407, DOI: 10.1080/14728214.2020.1820481 <a href="https://www.tandfonline.com/doi/full/10.1080/14728214.2020.1820481">https://www.tandfonline.com/doi/full/10.1080/14728214.2020.1820481</a></li>

</ol>
</section>
<section>
<ol>
<li>Fuermaier ABM, Tucha L, Butzbach M, Weisbrod M, Aschenbrenner S, Tucha O. ADHD at the workplace: ADHD symptoms, diagnostic status, and work-related functioning. J Neural Transm (Vienna). 2021 Jul;128(7):1021-1031. doi: 10.1007/s00702-021-02309-z. Epub 2021 Feb 2. PMID: 33528652; PMCID: PMC8295111. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8295111/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8295111/</a></li>

<li>Lu Liu et al. Deep learning model reveals potential risk genes for ADHD, especially Ephrin receptor gene EPHA5, Briefings in Bioinformatics, Volume 22, Issue 6, November 2021, bbab207, <a href="https://doi.org/10.1093/bib/bbab207">https://doi.org/10.1093/bib/bbab207</a>  <a href="https://academic.oup.com/bib/article/22/6/bbab207/6295376">https://academic.oup.com/bib/article/22/6/bbab207/6295376</a></li>

<li>Vidal, O.M., Vélez, J.I. &amp; Arcos-Burgos, M. ADGRL3 genomic variation implicated in neurogenesis and ADHD links functional effects to the incretin polypeptide GIP. Sci Rep 12, 15922 (2022). <a href="https://doi.org/10.1038/s41598-022-20343-z">https://doi.org/10.1038/s41598-022-20343-z</a>  <a href="https://www.nature.com/articles/s41598-022-20343-z">https://www.nature.com/articles/s41598-022-20343-z</a></li>

<li>Chan YS, Jang JT, Ho CS. Effects of physical exercise on children with attention deficit hyperactivity disorder. Biomed J. 2022 Apr;45(2):265-270. doi: 10.1016/j.bj.2021.11.011. Epub 2021 Nov 29. PMID: 34856393; PMCID: PMC9250090. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9250090/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9250090/</a></li>

<li>Cabana-Domínguez, J., Soler Artigas, M., Arribas, L. et al. Comprehensive analysis of omics data identifies relevant gene networks for Attention-Deficit/Hyperactivity Disorder (ADHD). Transl Psychiatry 12, 409 (2022). <a href="https://doi.org/10.1038/s41398-022-02182-8">https://doi.org/10.1038/s41398-022-02182-8</a>  <a href="https://www.nature.com/articles/s41398-022-02182-8">https://www.nature.com/articles/s41398-022-02182-8</a></li>

<li>Michelini G, Norman LJ, Shaw P, Loo SK. Treatment biomarkers for ADHD: Taking stock and moving forward. Transl Psychiatry. 2022 Oct 12;12(1):444. doi: 10.1038/s41398-022-02207-2. PMID: 36224169; PMCID: PMC9556670. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9556670/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9556670/</a></li>

</ol>
</section>
<section>
<ol>
<li>Lewis, K.J.S., Martin, J., Gregory, A.M. et al. Sleep disturbances in ADHD: investigating the contribution of polygenic liability for ADHD and sleep-related phenotypes. Eur Child Adolesc Psychiatry 32, 1253–1261 (2023). <a href="https://doi.org/10.1007/s00787-021-01931-2">https://doi.org/10.1007/s00787-021-01931-2</a>  <a href="https://link.springer.com/article/10.1007/s00787-021-01931-2">https://link.springer.com/article/10.1007/s00787-021-01931-2</a></li>

<li>Demontis, D., Walters, G.B., Athanasiadis, G. et al. Genome-wide analyses of ADHD identify 27 risk loci, refine the genetic architecture and implicate several cognitive domains. Nat Genet 55, 198–208 (2023). <a href="https://doi.org/10.1038/s41588-022-01285-8">https://doi.org/10.1038/s41588-022-01285-8</a>  <a href="https://www.nature.com/articles/s41588-022-01285-8">https://www.nature.com/articles/s41588-022-01285-8</a></li>

<li>Cabana-Domínguez, J., Llonga, N., Arribas, L. et al. Transcriptomic risk scores for attention deficit/hyperactivity disorder. Mol Psychiatry (2023). <a href="https://doi.org/10.1038/s41380-023-02200-1">https://doi.org/10.1038/s41380-023-02200-1</a>  <a href="https://www.nature.com/articles/s41380-023-02200-1">https://www.nature.com/articles/s41380-023-02200-1</a></li>

<li>Sophie Breunig, Jeremy M. Lawrence, Isabelle F. Foote, Hannah J. Gebhardt, Erik G. Willcutt, Andrew D. Grotzinger, Examining Differences in the Genetic and Functional Architecture of ADHD Diagnosed in Childhood and Adulthood medRxiv 2023.08.02.23293439; doi: <a href="https://doi.org/10.1101/2023.08.02.23293439">https://doi.org/10.1101/2023.08.02.23293439</a> <a href="https://www.medrxiv.org/content/10.1101/2023.08.02.23293439v1">https://www.medrxiv.org/content/10.1101/2023.08.02.23293439v1</a></li>

<li>Dengfeng Liu et al, Cry1Δ11 mutation induces ADHD-like symptoms through hyperactive dopamine D1 receptor signaling, JCI Insight. 2023;8(16):e170434, <a href="https://doi.org/10.1172/jci.insight.170434">https://doi.org/10.1172/jci.insight.170434</a>  <a href="https://insight.jci.org/articles/view/170434#SEC3">https://insight.jci.org/articles/view/170434#SEC3</a></li>

</ol>


</section>
</section>
<section>
<section id="slide-org9b8bf38">
<h2 id="org9b8bf38">Thank you for your Attention!</h2>
</section>
</section>
<section>
<section id="slide-org54fbdd5">
<h2 id="org54fbdd5">Glossary</h2>
<ol>
<li>Polygenic Risk Score: Score that is the effect of many genetic variants on an individual&rsquo;s phenotype, calculated as a weighted sum of trait-associated alleles</li>
<li>PBMC: peripheral blood cell with round nucleus</li>
<li>nsSNPs: non-synonymous single nucleotide polymorphism (change in single base -&gt; amino acid change)</li>
<li>Biomarker: objective measure that captures what is happening in a cell or an organism at a given moment</li>
<li>EEg -  electroencephalogram - test used to detect abnormalities in brain waves or electrical activity</li>
<li>Polygenic scores - individual index of genetic liability for a phenotype of interest and can be used to better understand the genetic relationship between ADHD and sleep problems.</li>
<li>ppt2 gene - palmitoyl protein thioesterase family</li>
<li>epigenome - changes to dna/histone in regulating gene expression</li>

</ol>

<style type="text/css">
.reveal section p {
    display: inline-block;
    font-size: 0.7em;
    line-height: 1.2em;
    vertical-align: top;
}

.reveal .title {
    font-size: 2.2em;
}

.reveal table {
  display: block;
  font-size: 0.5em;
  overflow: auto;
  max-height: 600px;
}

div.legalese {
  font-size: 1.4vh;
  position: fixed;
  left: 3px;
  bottom: 3px;
  z-index: 50;
}

.scrollbar {
  border: 1px solid #eee;
  height: 670px;
  overflow: auto;
  margin: 5px 0 0;
  padding: 5px 10px 10px 10px;
}
</style>
</section>
</section>
</div>
</div>
<script src="revjs/dist/reveal.js"></script>
<script src="revjs/plugin/markdown/markdown.js"></script>
<script src="revjs/plugin/notes/notes.js"></script>
<script src="revjs/plugin/search/search.js"></script>
<script src="revjs/plugin/zoom/zoom.js"></script>
<script>
// Full list of configuration options available here:
// https://github.com/hakimel/reveal.js#configuration
Reveal.initialize({

controls: true,
progress: false,
history: false,
center: true,
slideNumber: 'c',
rollingLinks: false,
keyboard: true,
mouseWheel: false,
fragmentInURL: false,
hashOneBasedIndex: false,
pdfSeparateFragments: true,
overview: true,
width: 1920,
height: 1080,
margin: 0.02,
minScale: 0.20,
maxScale: 1.00,

transition: 'convex',
transitionSpeed: 'default',
navigationMode: 'linear', controls: true,

// Plugins with reveal.js 4.x
plugins: [ RevealMarkdown, RevealNotes, RevealSearch, RevealZoom ],

// Optional libraries used to extend reveal.js
dependencies: [
]

});
</script>
</body>
</html>
